exchanges/exmo/exmo_types.go

```diff
@@ -139,14 +140,14 @@ type WalletHistory struct {
 	Begin   int64 `json:"begin,string"`
 	End     int64 `json:"end,string"`
 	History []struct {
-		Timestamp types.Time `json:"dt"`
-		Type      string     `json:"type"`
-		Currency  string     `json:"curr"`
-		Status    string     `json:"status"`
-		Provider  string     `json:"provider"`
-		Amount    float64    `json:"amount,string"`
-		Account   string     `json:"account"`
-		TXID      string     `json:"txid"`
+		Timestamp types.Time    `json:"dt"`
+		Type      string        `json:"type"`
+		Currency  currency.Code `json:"curr"`
+		Status    string        `json:"status"`
```

Comment:
Is this not order.Status?

Reply:
Nope this one is for wallet transactions